docs/cli/graphql-transformer/directives.md

```diff
@@ -476,70 +478,126 @@ type Post
 }
 ```
 
-Owner authorization specifies that a user can access an object. To
-do so, each object has an *ownerField* (by default "owner") that stores ownership information
-and is verified in various ways during resolver execution.
+<amplify-callout>
+
+Owner authorization requires an authentication type of **Amazon Cognito User Pools** to be enabled in your app.
+
+</amplify-callout>
+
+Owner authorization specifies whether a user can access or operate against an object. To do so, each object will get an `ownerField` field (by default `owner` will be added to the object if not specified) that stores ownership information and is verified in various ways during resolver execution.
 
-You can use the *operations* argument to specify which operations are augmented as follows:
+You can use the `operations` argument to specify which operations are enabled as follows:
 
 - **read**: If the record's owner is not the same as the logged in user (via `$ctx.identity.username`), throw `$util.unauthorized()` in any resolver that returns an object of this type.
-- **create**: Inject the logged in user's `$ctx.identity.username` as the *ownerField* automatically.
-- **update**: Add conditional update that checks the stored *ownerField* is the same as `$ctx.identity.username`.
-- **delete**: Add conditional update that checks the stored *ownerField* is the same as `$ctx.identity.username`.
+- **create**: Inject the logged in user's `$ctx.identity.username` as the `ownerField` automatically.
```

Comment:
I wonder if this is too much details. Do we need to have all the VTL info?

Reply:
I was wondering the same thing. I've updated to be clear without mentioning anything about $ctx or resolver language.